client/src/components/Dashboard/DashboardForecast/DashboardForecast.tsx

```diff
@@ -19,87 +17,84 @@ import Widget from 'components/Widget/Widget';
 
 // Utils
 import { capitalize, checkIfEnterOrSpacePressed } from 'utils/helpers';
+import {
+  WeatherType,
+  AirQualityType,
+  selectLocationWeather,
+  selectLocationAirQuality,
+  getLocationWeather,
+  getLocationAirQuality
+} from './DashboardForecast.utils';
 
 // Styles
 import { Flex } from 'styles/global';
 import * as S from '../Dashboard.styles';
 
-enum ForecastType {
-  hourly = 'hourly',
-  threeHourly = 'threeHourly',
-  daily = 'daily'
-}
+// Constants
+import { FORECAST_LABEL, AIR_QUALITY_LABEL } from '../../../constants';
 
 const DashboardForecast: FunctionComponent = () => {
-  const [infoType, setInfoType] = useState<string>('forecast');
-  const [selectedForecastType, setSelectedForecastType] = useState<string>(ForecastType.hourly);
   const [carouselChildPointerEv, setCarouselChildPointerEv] = useState<boolean>(false);
+  const [infoType, setInfoType] = useState<string>(FORECAST_LABEL);
+  const [selectedForecastType, setSelectedForecastType] = useState<WeatherType | AirQualityType>(
```

Comment:
have you tried to create separate shared type?
`type ForecastType = WeatherType | AirQualityType`